### DISCUSSION TITLE
Can it support .net4.6?

### DISCUSSION THREAD

**Discussion by xmzzy** [icsharpcode/SharpZipLib#760] [Q&A] [answered]
### Steps to reproduce
1.
2.
3.

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### Obtained from (only keep the relevant lines)
- Compiled from source, commit: _______
- Downloaded from GitHub
- Package installed using NuGet

**Comment by david-beckman** [comment, marked as answer]
As per [ICSharpCode.SharpZipLib.csproj](https://github.com/icsharpcode/SharpZipLib/blob/master/src/ICSharpCode.SharpZipLib/ICSharpCode.SharpZipLib.csproj), this is targeting .NET Standard 2.0, 2.1, and .NET Framework 4.5. Per [the .NET Standard documentation](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) .NET Framework 4.6.1 and 4.6.2 can use .NET Standard 2.0 assemblies.

**Comment by christophwille** [comment]
https://devblogs.microsoft.com/dotnet/net-framework-4-5-2-4-6-4-6-1-will-reach-end-of-support-on-april-26-2022/
